### DISCUSSION TITLE
Allow to make network configuration mandatory

### DISCUSSION THREAD

**Discussion by champtar** [rhinstaller/anaconda#3958] [Ideas]
I'm building an appliance ISO installing an rpm-ostree system.
This appliance doesn't make sense without network but there is no clean/easy way to make the Network menu mandatory.

For now I'm patching `usr/lib64/python3.6/site-packages/pyanaconda/ui/{tui,gui}/spokes/network.py`
and put that into `product.img`, but I would love if there was a boolean in the anaconda product conf file.

**Comment by poncovka** [comment]
Could you post here your patch to get a better idea of the feature, please?
